scripts/tools/RegisterPublishedDeliveryWebhook.php

```diff
@@ -0,0 +1,131 @@
+<?php
+
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2021 (original work) Open Assessment Technologies SA;
+ */
+
+declare(strict_types=1);
+
+namespace oat\taoPublishing\scripts\tools;
+
+
+use Exception;
+use oat\oatbox\event\EventManager;
+use oat\oatbox\extension\script\ScriptAction;
+use oat\oatbox\reporting\Report;
+use oat\oatbox\service\ConfigurableService;
+use oat\oatbox\service\ServiceManagerAwareTrait;
+use oat\tao\model\webhooks\configEntity\Webhook;
+use oat\tao\model\webhooks\WebhookEventsService;
+use oat\tao\model\webhooks\WebhookRegistryManager;
+use oat\tao\model\webhooks\WebhookRegistryManagerInterface;
+use oat\taoPublishing\model\publishing\event\RemoteDeliveryCreatedEvent;
+
+class RegisterPublishedDeliveryWebhook extends ScriptAction
```

Comment:
do we need this inside the manifest ?
also do we need a migration for this?

Reply:
No this is command on demand. We may not want to have this webhook registered.

Reply:
what is the action that OPS have to do in order to enable this ?
how will this play on prod ?
do you think it might be easier to add a value in .env then to run some specific command that is not documented anywhere ?

Reply:
No I don't think we can use env file. We are not creating any additional behaviour here that we can attach feature flag. 
By adding config entries we are attaching webhook with provided configuration to desired event. 
If we wish to add this to prod we can either simply run command or using client environment extension add install script with desired configuration to manifest file.